File: ejercicios/c++/ejercicio050.cpp
#include<iostream>

using namespace std;

int main() {

	int num_a; 
	int num_b;
	int copia_a;

	int cont = 0;

	cout << "ingrese el primer numero: ";
	cin >> num_a;
	
	cout << "ingrese el segundo numero: ";
	cin >> num_b;
	
	copia_a = num_a;
	
	while(copia_a - num_b > 0){
		cont++;
		copia_a-=num_b;
	}
		
	cout << num_a << " / " << num_b << " = " << cont << endl;
		
	return 0;

}

File: ejercicios/python/ejercicio050.py
num_a = int(input("ingrese el primer numero: "))
num_b = int(input("ingrese el segundo numero: "))

copia_a = num_a

cont = 0

while copia_a - num_b > 0:
	cont = cont + 1
	copia_a = copia_a - num_b
	
print("%i / %i = %i" %(num_a, num_b, cont))
	
